src/components/ModalForm.js

```diff
@@ -19,101 +18,127 @@ export default function ModalForm(props) {
+            <Form.Group className="mb-3">
+              <Form.Label>Name</Form.Label>
+              <Form.Control
                 type="text"
+                placeholder="Recipe Name"
                 value={recipeName}
-                onChange={(event) => setRecipeName(event.target.value)}
+                onChange={(e) => setRecipeName(e.target.value)}
               />
-            </label>
-            <br />
-            <label>Ingredients: </label>
-            <input
-              value={ingredient}
-              onChange={(e) => setIngredient(e.target.value)}
-            />
-            <button
-              onClick={() => {
-                updateIngredients([
-                  ...ingredients,
-                  { id: ingredients.length, name: ingredient },
-                ]);
-              }}
-              type="button"
-            >
-              Add
-            </button>
-            <ul>
-              {ingredients.map((ingredient) => (
-                <li key={ingredient.id}>
-                  {ingredient.name}
-                  <button
-                    onClick={() => {
-                      updateIngredients(
-                        ingredients.filter((a) => a.id !== ingredient.id)
-                      );
-                    }}
-                  >
-                    Delete
-                  </button>
-                </li>
-              ))}
-            </ul>
-            <label>
-              Min Time:
-              <input
-                type="number"
-                value={minTime}
-                onChange={(event) => updateMinTime(event.target.value)}
-              />
-            </label>
-            <br />
-            <label>
-              Max Time:
-              <input
-                type="number"
-                value={maxTime}
-                onChange={(event) => updateMaxTime(event.target.value)}
-              />
-            </label>
-            <br />
-            <label>
-              Created By:
-              <input
+            </Form.Group>
+            <Form.Group>
+              <Form.Label>Ingredients</Form.Label>
+              <InputGroup className="mb-3">
+                <Form.Control
+                  type="text"
+                  placeholder="Ingredient Name"
+                  onChange={(e) => setIngredient(e.target.value)}
+                />
+                <Button
+                  onClick={() => {
+                    updateIngredients([
+                      ...ingredients,
+                      { id: ingredients.length, name: ingredient },
+                    ]);
+                  }}
+                  type="button"
+                  variant="outline-dark"
+                >
+                  Add
+                </Button>
+              </InputGroup>
+              <ul>
+                {ingredients.map((ingredient) => (
+                  <li key={ingredient.id}>
+                    {ingredient.name}
+                    <Button
```

Comment:
you may want to add a margin-left to the button so it's not plastered next to the ingredient name.

That or wrap ingredient name in a span so there's some auto spacing

Reply:
done